<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeZoneConfigurationWindowForm</class>
 <widget class="QWidget" name="timeZoneConfigurationWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>398</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="leftSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Time zone</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="rightSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabTimeZone">
      <attribute name="title">
       <string>Time zone</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutTimeZone">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxTimeZone">
         <property name="title">
          <string>Time zone</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutCurrent">
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="labelTimeZoneInfo">
            <property name="text">
             <string>By default, the time displayed in Stellarium is interpreted as the local time in the system's time zone, not in the displayed location's time zone. This behaviour can be changed by changing Stellarium's global time zone settings.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="radioButtonLocalSettings">
            <property name="text">
             <string>Use system settings (default)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QRadioButton" name="radioButtonUtc">
            <property name="text">
             <string>Universal Coordinated Time (UTC)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButtonOffset">
            <property name="text">
             <string>Offset from UTC (hours):</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxOffset">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-24.000000000000000</double>
            </property>
            <property name="maximum">
             <double>24.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.250000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="labelOffset">
            <property name="text">
             <string>(offset = local time - UTC time)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="radioButtonUserDefined">
            <property name="text">
             <string>User-defined:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QFrame" name="frameUserDefined">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lineEditUserDefined"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonEditTimeZone">
               <property name="text">
                <string notr="true">...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonSave">
            <property name="text">
             <string>Save settings</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QLabel" name="labelRestart">
            <property name="text">
             <string>Any changes will take effect the next time Stellarium is started.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <spacer name="verticalSpacerTimeZone">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDisplayFormats">
      <attribute name="title">
       <string>Display formats</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutTabTimeFormat">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxDisplayFormats">
         <property name="title">
          <string>Display formats</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelDisplayFormats">
            <property name="text">
             <string>These settings control the way time and date are displayed in the bottom bar.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxTimeFormat">
            <property name="title">
             <string>Time display format</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayoutTimeFormat">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButtonTimeDefault">
               <property name="text">
                <string>System default</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonTime12Hour">
               <property name="text">
                <string>12-hour format</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonTime24Hour">
               <property name="text">
                <string>24-hour format</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxDateFormat">
            <property name="title">
             <string>Date display format</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayoutDateFormat">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButtonDateDefault">
               <property name="text">
                <string>System default</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonDateYMD">
               <property name="text">
                <string>yyyy-mm-dd (ISO 8601)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonDateDMY">
               <property name="text">
                <string>dd-mm-yyyy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonDateMDY">
               <property name="text">
                <string>mm-dd-yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerDisplayFormats">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAbout">
      <attribute name="title">
       <string>About plugin</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelTitle">
         <property name="styleSheet">
          <string notr="true">QLabel {
	font: bold 18pt ;
}</string>
         </property>
         <property name="text">
          <string notr="true">Time Zone plug-in</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelVersion">
         <property name="text">
          <string notr="true">Version %1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCopyright">
         <property name="text">
          <string notr="true">Copyright &amp;copy; 2010 Bogdan Marinov</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLicense">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later version.

This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.

You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>radioButtonLocalSettings</tabstop>
  <tabstop>radioButtonUtc</tabstop>
  <tabstop>radioButtonOffset</tabstop>
  <tabstop>doubleSpinBoxOffset</tabstop>
  <tabstop>radioButtonUserDefined</tabstop>
  <tabstop>lineEditUserDefined</tabstop>
  <tabstop>pushButtonEditTimeZone</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>radioButtonTimeDefault</tabstop>
  <tabstop>radioButtonTime12Hour</tabstop>
  <tabstop>radioButtonTime24Hour</tabstop>
  <tabstop>radioButtonDateDefault</tabstop>
  <tabstop>radioButtonDateYMD</tabstop>
  <tabstop>radioButtonDateDMY</tabstop>
  <tabstop>radioButtonDateMDY</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonUserDefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameUserDefined</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
